### ISSUE TITLE
Make startup nicer

### ISSUE THREAD

**Issue by makenai** [makenai/robotnik#5]
It's not friendly to just start a server and ask someone to navigate to it. It would be nicer if it was packaged up like one of these:

https://nodesource.com/blog/node-desktop-applications

**Comment by BrianGenisio** [comment]
Yes!  This is a requirement for making the platform accessible.  You'd probably need to explicitly support a lot of things, like different IO layers so that it is easy to get working without them needing to `npm install` things... but I totally agree... a "captive runtime" install has to be in the roadmap.
